services/net/notification/NotificationManager.cs

```diff
@@ -379,6 +380,40 @@ private async Task ResetAlertAsync(API.Areas.Services.Models.Content.ContentMode
         }
     }
 
+    /// <summary>
+    /// Get all the subscribers for the specified 'notification' and 'content'.
+    /// </summary>
+    /// <param name="notification"></param>
+    /// <param name="content"></param>
+    /// <returns></returns>
+    private async Task<IEnumerable<API.Areas.Services.Models.Notification.UserModel>> GetNotificationSubscribersAsync(API.Areas.Services.Models.Notification.NotificationModel notification, API.Areas.Services.Models.Content.ContentModel content)
```

Comment:
I was unable to duplicate subscriber issues, but there was a bug with distribution.  I have also moved this logic from the NotificationValidator to here.